ironfish/src/rpc/routes/wallet/getTransactions.ts

```diff
@@ -99,7 +103,17 @@ router.register<typeof GetAccountTransactionsRequestSchema, GetAccountTransactio
     let count = 0
     let offset = 0
 
-    for await (const transaction of account.getTransactionsByTime()) {
+    if (request.data.sequence !== undefined && request.data.sequence < GENESIS_BLOCK_SEQUENCE) {
+      throw new ValidationError('Sequence must be a positive number')
+    }
+
+    let streamFn = () => account.getTransactionsByTime()
```

Comment:
This might be the one time I suggest a ternary because you also don't need to create a closure here.

```ts
const stream = request.data.sequence
      ? account.getTransactionsBySequence(request.data.sequence)
      : account.getTransactionsByTime()
```

Reply:
Yep, for sure. Updated.